dot/sync/syncer_integeration_test.go

```diff
@@ -62,24 +62,21 @@ func newMockNetwork() *mocks.Network {
 	return m
 }
 
-//go:generate mockgen -destination=mock_telemetry_test.go -package $GOPACKAGE github.com/ChainSafe/gossamer/dot/telemetry Client
-
```

Comment:
Why is this removed? Don't we need it? I can't find another one in the `dot/sync` package.

Reply:
This shouldn't have been removed, I've re-added.